k8s/cert-manager/cluster-issuer.yml

```
@@ -0,0 +1,14 @@
+metadata:
+  name: letsencrypt-prod
+spec:
+  acme:
+    server: https://acme-v02.api.letsencrypt.org/directory
+    email: reconpoint@example.com
+    privateKeySecretRef:
+      name: letsencrypt-prod
+    solvers:
+    - http01:
+        ingress:
+          class: nginx
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)